production/user_guide.md

```diff
@@ -175,7 +176,62 @@ As shown above, in the field labeled "Resource types requested," type "Coverage.
 
 If you’d like to use the command line or implement this API call in code, look in the `Curl` section (shown in the image above) for the request you just made. Not far below that, you can see the response: an `HTTP 202 Accepted` giving a link in the content-location header for status information on your Coverage job.
 
-### 5. Getting your data
+### 5. Using the `_since` parameter
```

Comment:
Same recommendations on this page as above.  (Global)

Reply:
Fixed, resolved.